e2e/cypress/tests/integration/messaging/collapsed_message_spec.js

```diff
@@ -49,15 +49,9 @@ describe('Long message', () => {
 function verifyCollapsedPost() {
     // * Verify show more button
     cy.get('#showMoreButton').scrollIntoView().should('be.visible').and('have.text', 'Show more');
-
-    // * Verify gradient
-    cy.get('#collapseGradient').should('be.visible');
```

Comment:
@jgilliam17 Because of this change to how we're doing this fade out, there's no longer a `collapseGradient` element that we can look for to verify that the fade out effect is present. It's just a natural part of one of the existing CSS classes on the post text that's added for overflowing posts